README.md

```diff
@@ -11,88 +11,84 @@
 
 [![Gradle](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/built-with/gradle_vector.svg)](https://gradle.org/)
 [![Java](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/built-with/java17_vector.svg)](https://www.java.com/)
-
 </div>
 
-# Information's
+# Welcome to EternalCore! 🚀
 
-- EternalCore supports Minecraft latest minor version of each major version starting from 1.17+, eg. 1.17, 1.18, 1.19,
-  etc.
-- EternalCore requires Java 17 or higher. On older versions, the plugin may not work properly.
+EternalCore is your ultimate companion for enhancing your Minecraft server experience. 💎
 
-# Development builds
+## :information_source: Information
 
-Get the development builds from [GitHub Actions](https://github.com/EternalCodeTeam/EternalCore/actions)
+- EternalCore fully supports Minecraft's latest minor versions starting from 1.17 onwards, including 1.18 and beyond!
+- It requires **Java 17 or higher** to operate optimally. For older versions of Java, the plugin's performance might be affected.
+- If you have any questions, perhaps you will find a solution to them in our [documentation](https://docs.eternalcode.pl/eternalcore/introduction.html), you can also ask us about it on [discord](https://discord.gg/FQ7jmGBd6c).
 
-<details><summary>Video guide</summary>
+## :hammer_and_wrench: Development Builds
 
-![gif](https://i.imgur.com/hmv38VS.gif)
+Get the latest development builds from our [GitHub Actions](https://github.com/EternalCodeTeam/EternalCore/actions).
 
+<details><summary>🎥 Video Guide</summary>
+  <img src="https://i.imgur.com/hmv38VS.gif" alt="Video Guide">
 </details>
 
-# Features
-
-- Slot Bypass `(eternalcore.bypass.slot)`
-- AFK system
-- Chat features, including:
-    - Admin chat
-    - Auto Messages system `(with sequence or random option and command to disable receiving message)`
-    - Chat on/off switch
-    - Chat slow mode
-    - /ignore /unignore `(with -all option)` command
-    - /msg, /socialspy /reply command
-    - /helpop command
-- Open utility blocks with simple commands, such as `/workbench`
-- Player inventory viewer
-- Player attribute management, including heal, feed, teleportation, and godmode modification
-- Ping command to ensure client <-> server connection
-- Player information command `(/whois)`
-- Home, warp, and spawn system
-- PlaceholderAPI support
-- Customizable and translatable messages `(Optional player language selection)`
-- Advanced configuration system for customizing almost every aspect of the plugin
-- Database integration: `(PostGreSQL, SQLite, MySQL, MariaDB, H2)`
-- Adventure and [MiniMessage](https://docs.advntr.dev/minimessage/format.html) integration with legacy color
-  processing `(e.g &7, &e)` and message destination specification
-- And much more!
-
-# Building
-
-To build EternalCore do the following (Make sure you have JDK 17 or higher)
-
-```
+## :sparkles: Features
```

Comment:
Można by dodać trochę więcej rzeczy. Wierze, że mamy dużo jeszcze fajnych rzeczy, ogólnie jakby ktoś szukał pluginu na rozwiązanie problemu A to taki szybki przegląd z listy jest spoko pod ctrl + f, a tak to trzeba gdzieś tam na stronkę wbijać

Reply:
Jakieś sugestie?

Reply:
Nie mam kompa ale na docksach jest sporo contentu